airbyte-ci/connectors/connector_ops/pyproject.toml

```diff
@@ -17,7 +17,7 @@ GitPython = "^3.1.29"
 pydantic = "^1.9"
 PyGithub = "^1.58.0"
 rich = "^13.0.0"
-pydash = "^7.0.4"
+pydash = "^6.0.2"
```

Comment:
Why downgrade?

Reply:
`connectors_qa` depends on two other internal packages which are using `pydash` but on different version:
* `medata_service/lib` uses `6`
* `connector_ops` uses `7`

 I'm aligning to  `metadata_service/lib` version because this package is also a dependency of `metadata_service/orchestrator`, which is also declaring a dependency on `pydash`.

Aligning to `6` makes me modifying a single package (`connector_ops`) instead of 2 (`metadata_service/lib` / `metadata_service/orchestrator`). A part from that I don't think there's a good reason to stay on `6`.